solutions/standard/variables.tf

```diff
@@ -0,0 +1,142 @@
+  default     = "us-south"
+}
+
+########################################################################################################################
+# Secrets Manager
+########################################################################################################################
+
+variable "secrets_manager_instance_name" {
+  type        = string
+  description = "The name to give the Secrets Manager instance that will be provisioned by this solution."
+  default     = "base-security-services-sm"
+}
+
+variable "service_plan" {
+  type        = string
+  description = "The service/pricing plan to use when provisioning a new Secrets Manager instance. Allowed values: 'standard' and 'trial'. Only used if `provision_sm_instance` is set to true."
+  default     = "standard"
+  validation {
+    condition     = contains(["standard", "trial"], var.service_plan)
+    error_message = "Allowed values for sm_service_plan are \"standard\" and \"trial\"."
+  }
+}
+
+variable "service_endpoints" {
+  # public-and-private until IBM Console connects to SM via private endpoints
+  type        = string
+  description = "The service endpoints to enable for all services deployed by this solution. Allowed values are `private` or `public-and-private`. If selecting `public-and-private`, communication to the instances will all be done over the public endpoints. Ensure to enable virtual routing and forwarding (VRF) in your account if using `private`, and that the terraform runtime has access to the the IBM Cloud private network."
+  default     = "public-and-private"
+  validation {
+    condition     = contains(["private", "public-and-private"], var.service_endpoints)
+    error_message = "The specified service_endpoints is not a valid selection. Allowed values are `private` or `public-and-private`."
+  }
+}
+
+variable "secret_manager_tags" {
+  type        = list(any)
+  description = "The list of resource tags that you want to associate with your Secrets Manager instance."
+  default     = []
+}
+
+########################################################################################################################
+# Key Protect
+########################################################################################################################
+
+variable "existing_kms_instance_guid" {
+  type        = string
+  description = "The CRN of an existing KMS instance to use. Only needed if creating kms auth policy and if not providing an existing kmd key crn"
+  nullable    = true
+  default     = null
+}
+
+variable "skip_kms_iam_authorization_policy" {
+  type        = bool
+  description = "Set to true to skip the creation of an IAM authorization policy that permits all Secrets Manager instances in the resource group to read the encryption key from the KMS instance. If set to false, pass in a value for the KMS instance in the existing_kms_instance_guid variable. In addition, no policy is created if kms_encryption_enabled is set to false."
```

Comment:
Remove the last sentence from the description here. `kms_encryption_enabled` is not a variable we expose in the DA since we hard code it to true